waspc/packages/ts-inspect/README.md

```diff
@@ -0,0 +1,26 @@
+NOTE: `typescript` is purposefully a normal dependency instead of a dev
+dependency.
+
+Run the program `node ./dist/index.js` and pass a list of export requests over
+stdin:
+
+```json
+[
+  { "filenames": ["./src/exports.ts"] },
+  {
+    "tsconfig": "~/dev/wasp-todoapp/src/client/tsconfig.json",
+    "filenames": "~/dev/wasp-todoapp/src/client/MainPage.tsx"
```

Comment:
@craigmc08 I'm a bit confused about the API here.

There is a key called `filenames` but it only has a single string? Is this a mistake in writing the README?

Reply:
That's a mistake in the README, should be an array. Fixed!